feat: interface health anomaly fields and term

rahul-kriplani-jnpr opened the pull request:
- in-mbps
- out-mbps
- optical-temp
- optics-tx-power
- optics-rx-power

rahul-kriplani-jnpr committed:
feat: interface health anomaly fields and term

- in-mbps
- out-mbps
- optical-temp
- optics-tx-power
- optics-rx-power

rahul-kriplani-jnpr commented on juniper_official/interface/check-interface-in-out-errors-traffic-state-flaps.rule:
```diff
@@ -224,17 +242,33 @@ healthbot {
                  * Sets color to green when in-util is below low-threshold
                  */				
-                term interface-in-traffic-normal {
+                term is-interface-in-traffic-normal {
```
I have made the term name consistent with others in the trigger

rahul-kriplani-jnpr commented:
Have added the new terms by taking the reference provided.

I had a doubt, is this term required?
https://github.com/Juniper/healthbot-rules/blob/1548b46c104614909ad5627b007657988305a11a/juniper_official/chassis/check-routing-engine-temperature.rule#L167
aren't we already checking this in
https://github.com/Juniper/healthbot-rules/blob/1548b46c104614909ad5627b007657988305a11a/juniper_official/chassis/check-routing-engine-temperature.rule#L135
is this to handle when jaiml feature is not enabled and we don't get anomaly status?

rahul-kriplani-jnpr commented:
reference to the closed MR: https://github.com/Juniper/healthbot-rules/pull/1009

rahul-kriplani-jnpr commented on juniper_official/interface/check-interface-in-out-errors-traffic-state-flaps.rule:
```diff
@@ -224,17 +242,33 @@ healthbot {
+                term is-interface-in-traffic-anomalous {
+                    when {
+                        less-than "$in-util" "$low-threshold";
+                        equal-to "$in-mbps-anomaly" 1;
```
the anomaly status will not be greater than one. Should we still add any condition to handle values greater than 1?

gaddevijay replied:
Yes. this is required

rahul-kriplani-jnpr replied:
at this point AIOPs doesn't have anomaly > 1

rahul-kriplani-jnpr commented:
> Have added the new terms by taking the reference provided.
> 
> I had a doubt, is this term required?
> 
> https://github.com/Juniper/healthbot-rules/blob/1548b46c104614909ad5627b007657988305a11a/juniper_official/chassis/check-routing-engine-temperature.rule#L167
> 
> 
> aren't we already checking this in
> https://github.com/Juniper/healthbot-rules/blob/1548b46c104614909ad5627b007657988305a11a/juniper_official/chassis/check-routing-engine-temperature.rule#L135
> 
> 
> is this to handle when jaiml feature is not enabled and we don't get anomaly status?

I believe this is required for the triggers I have updated, right? @gaddevijay

rahul-kriplani-jnpr committed:
feat: add normal term without aiops anomaly check

rahul-kriplani-jnpr commented:
> > Have added the new terms by taking the reference provided.
> > I had a doubt, is this term required?
> > https://github.com/Juniper/healthbot-rules/blob/1548b46c104614909ad5627b007657988305a11a/juniper_official/chassis/check-routing-engine-temperature.rule#L167
> > 
> > aren't we already checking this in
> > https://github.com/Juniper/healthbot-rules/blob/1548b46c104614909ad5627b007657988305a11a/juniper_official/chassis/check-routing-engine-temperature.rule#L135
> > 
> > is this to handle when jaiml feature is not enabled and we don't get anomaly status?
> 
> I believe this is required for the triggers I have updated, right? @gaddevijay

updated this for the interfaces rules.
